src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Lottery {
+    public String getRandomBall() {
+        StringBuilder balls = new StringBuilder();
+
+        for (int i = 0; i < 3; i++) {
+            balls.append(new Ball().toString()).append("\n");
+        }
+
+        return balls.toString();
```

Comment:
here we should create a Ball object with a random color and random number
Example:
```
...
return new Ball(colorSupplier.getRandomColor(), random.nextInt(UPPER_BOUND));
```